File: Java/905. Sort Array By Parity.java
class Solution {
    public int[] sortArrayByParity(int[] nums) {
        int i =0;
        for (int j=0; j<nums.length;j++){
            if (nums[j]%2 == 0){
                int temp = nums[j];
                nums[j] = nums[i];
                nums[i] = temp;
                i++;
            }
        }
        return nums;
    }
}

File: Python/905. Sort Array By Parity.py
class Solution:
    def sortArrayByParity(self, nums: List[int]) -> List[int]:
        if len(nums)<=1:
            return nums
        i = 0
        for j in range(len(nums)):
            if (nums[j] % 2 == 0):
                nums[i], nums[j] = nums[j], nums[i]
                i+=1
        return nums